tests/func/test_data_cloud.py

```diff
@@ -588,3 +588,23 @@ def test_pull_00_prefix(tmp_dir, dvc, remote, monkeypatch):
     stats = dvc.pull()
     assert stats["fetched"] == 2
     assert set(stats["added"]) == {"foo", "bar"}
+
+
+@pytest.mark.parametrize("remote", full_clouds, indirect=True)
+def test_pull_no_00_prefix(tmp_dir, dvc, remote, monkeypatch):
```

Comment:
What this does is, it is triggering a prefix based search while that prefix doesn't exist (unlike `test_pull_00_prefix`, where it is triggering a prefix based search while the prefix exists).